crates/core_simd/src/masks.rs

```diff
@@ -507,58 +507,58 @@ where
     }
 }
 
-/// Vector of eight 8-bit masks
+/// A 64-bit SIMD vector mask for eight elements of 8 bits.
```

Comment:
Masks have unspecified size (some architectures `mask16x8` is 128-bits, other architectures it is an 8-bit bitmask, we may also get architectures where it isn't either of those (e.g. masks could always have 8-bit elements so `mask8x8`, `mask16x8`, ..., and `mask64x8` would all be 64-bits)).

These comments need to not state a specific size for masks:
```suggestion
/// A SIMD vector mask of eight elements, for operating with SIMD vectors with 8 bit elements. 
```

Reply:
Ah yeah, that makes sense. Thanks everyone! I'll also update the `Mask` documentation to note that these are *boolean* masks -- I got tricked thinking of bitwise masking operations.

Reply:
assuming you'll change the other comments too...

Reply:
Yeah, pondering the right wording. Don't worry :+1: